packages/babel-preset-sui/lib/index.js

```diff
@@ -1,12 +1,41 @@
-const isInstalled = require('./is-installed')
-const cleanList = require('./clean-list')
+console.log('new babel')
 
-module.exports = {
-  presets: cleanList([
+function plugins(api, opts) {
```

Comment:
Are we crazy enough ?! 
https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining#via-babelrc-recommended

Reply:
hahaha let's do it as a new feature as soon as we have babel7. :)